### DISCUSSION TITLE
Rename "New Display Folder" action to "Add to Display Folder"

### DISCUSSION THREAD

**Discussion by edhans** [TabularEditor/TabularEditor3#1064] [Ideas]
this is v3.10.

If I right-click on a display folder and select "New Measure" the new measure appears under that display folder. That seems correct.

If I right-click on a display folder and select "New Display folder" the folder I rightclicked on is moved underneath the new folder I am creating. It seems it should create the new folder _under_ the folder I right-clicked on.

**Comment by otykier** [comment]
This is by design. I see your point but the problem is that we can’t create empty display folders in the TOM, since display folders are just properties of measures, columns and hierarchies. We could change the wording to something like “Add to display folder” instead of “Create display folder”. Would that be better?

**Comment by edhans** [comment]
I suppose. Yeah, if the model has that restriction not much you can do. At least that would prevent the inadvertent moving. The pattern then should be "Create the thing you want" then "Add to display folder".

Thanks!
